lib/resources/dstu2/soarian_allergy_intolerance.yaml

```diff
@@ -0,0 +1,346 @@
+---
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: AllergyIntolerance
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
```

Comment:
remove all non-binding related fields in this file. This service doesn't allow update/modify, so these aren't being used.